app/src/pages/inside/projectSettingsPageContainer/content/demoDataContent/demoDataTab.scss

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.demo-data-tab {
+  display: flex;
+  flex-direction: column;
+
+  .description-details {
+    font-family: $FONT-ROBOTO-REGULAR;
+    font-size: 13px;
+    color: $COLOR--almost-black;
+    line-height: 20px;
+    text-align: left;
+    padding-bottom: 23px;
+    max-width: 660px;
+    @media (max-width: $SCREEN_XS_MAX) {
+      width: 100%;
+    }
+  }
+
+  .description-delimiter {
+    width: 100%;
+    height: 1px;
+    background: $COLOR--e-100;
+    border: none;
+    margin: 0;
+  }
+
+  .description-header {
+    color: $COLOR--almost-black;
+    font-size: 13px;
+    font-family: $FONT-ROBOTO-MEDIUM;
+    line-height: 1.5;
+    text-indent: 1px;
+    text-align: left;
+    margin-top: 24px;
+    margin-bottom: 6px;
+    @media (max-width: $SCREEN_XS_MAX) {
+      margin-top: 12px;
+      margin-bottom: 5px;
+    }
+  }
+}
+
+.description-list {
+  margin: 10px 0 24px 26px;
+  list-style-type: none;
+
+  li {
+    margin: 0 0 3px 0;
+    padding-left: 8px;
+    color: $COLOR--charcoal-grey;
+    font-size: 13px;
+    line-height: 1.5;
+  }
+
+  li::before {
+    content: '\2022';
+    color: $COLOR--topaz-2;
+    font-weight: bold;
+    display: inline-block;
+    width: 1em;
+    margin-left: -1em;
```

Comment:
Since you set padding-left: 8px above
you can remove all except content and color
add position absolute, left: 0, width: 4px
margin-left: -1em looks unclear and unnecessary at all